manifests/params.pp

```diff
@@ -11,7 +11,8 @@
   $config_defaults       = {}
   $config_dir            = '/etc/consul'
   $config_hash           = {}
-  $config_mode           = '0660'
+  $config_mode           = '0664'
```

Comment:
this will break container volume mounts as the container itself does not share the users on the host, and thus counts as "other users".  It was set as read-only to prevent tweaking.  Keep in mind that in a container scenario the surface area of attack will be the containers themselves (that hosts all open ports).  I can add a "config_mode_real" so it can be flipped back and forth based on the deployment mode?